miwa.rb

```diff
@@ -0,0 +1,45 @@
+class Indicatory
+    puts 'Щосаме вам потрібно порахувати ?'
+    answer = gets.chomp.upcase 
+    if answer == 'Electric'
```

Comment:
We can store all possible values in const at the top of this class.

Reply:
And then check for it like `answer.in?(CONST)`. The const is hash with name of product as key and class as value.